README.md

```diff
@@ -98,6 +98,20 @@ You can run `npm test` to execute the test suite and linters. To help you develo
 ### The examples
 The examples are hosted within the docs folder and are ran in the simple add that loads the Datepicker. To extend the examples with a new example, you can simply duplicate one of the existing examples and change the unique properties of your example.
 
+## Accessibility
+
+### Keyboard support
+
+* *Left*: Move to the previous day.
+* *Right*: Move to the next day.
+* *Up*: Move to the previous week.
+* *Down*: Move to the next week.
+* *PgUp*: Move to the previous month.
+* *PgDn*: Move to the next month.
+* *Home*: Move to the previous year.
+* *End*: Move to the next year.
+* *Enter/Esc/Tab*: close the calendar. (Enter & Esc calls preventDefautl)
+
```

Comment:
i believe the reason for this fork to exist is captured here:

https://github.com/concur/react-datepicker/commit/c775b3152e5fcc1e5f65a40ade34f42d53426a2e

our fork _allows_ for invalid dates to be used

I'm not sure if we retained that in the merge here.